init.rb

```diff
@@ -37,7 +37,7 @@
            }
 
   menu :admin_menu, :redmine_issue_templates, { controller: 'global_issue_templates', action: 'index' },
-       caption: :global_issue_templates
+       caption: :global_issue_templates, :html => { :class => 'icon icon-global_issue_templates' }
```

Comment:
__[rubocop]__
* Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

<!-- Comment by [SideCI](https://sideci.com) -->